tests/xtimer_usleep/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       xtimer_usleep test application
+ *
+ * @author      Francisco Acosta <francisco.acosta@inira.fr>
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "xtimer.h"
+#include "timex.h"
+
+#define ONE_SEC_SLEEP     (1 * SEC_IN_USEC)
+#define FIVE_SEC_SLEEP    (5 * SEC_IN_USEC)
+#define TEN_SEC_SLEEP     (10 * SEC_IN_USEC)
+
+int main(void)
+{
+    printf("This test will print \"Slept for X sec...\" every 1, 5 and 10 seconds.\n");
+    printf("\n<======== If using pyterm, this is the time when started.\n\n");
+
+    xtimer_usleep(ONE_SEC_SLEEP);
+    printf("Slept for 1 sec...\n");
```

Comment:
Why not display the message before xtimer_usleep. With the message "Sleeping during x s" ?